### PR TITLE
Update spanish localization

### PR DESCRIPTION
**Pull request by AFROM**
In 100% :D

### CODE REVIEW

**Comment by Sarjuuk** [comment]
Wow, thanks man.

Just one thing. The square brackets were just meant to mark untranslated strings. If you remove them, I'll merge this file.

I'd also like to request another thing or two. The localisation is split in server side and client side. You took care of the server part. `static/js/locale_eses.js` has some custom parts, that I was unable to translate. They are tagged with "custom" . If you could also take care of these, that would be awesome.

Lastly there was a translation problem within the game itself. But can't get an example on mobile. I'll create a ticket, when I get home.

**Comment by AFROM** [comment]
Perfect, i'll take a look to that file :+1:

File: localization/locale_eses.php [hunk 1]
```diff
@@ -763,12 +763,12 @@
...
         'costPerSec'    => ", mas %s por segundo",
         'costPerLevel'  => ", mas %s por nivel",
-        'stackGroup'    => "[Stack Group]",
-        'linkedWith'    => "[Linked with]",
-        '_scaling'      => "[Scaling]",
+        'stackGroup'    => "Grupo de aplilamiento",
+        'linkedWith'    => "Asociado con",
+        '_scaling'      => "Escala",
         'scaling'       => array(
-            'directSP' => "[+%.2f%% of spell power to direct component]",        'directAP' => "[+%.2f%% of attack power to direct component]",
```
Comment (Sarjuuk) [inline localization/locale_eses.php]: same here

File: localization/locale_eses.php [hunk 2]
```diff
@@ -934,9 +934,9 @@
             "meta",                 "roja(s)",          "amarilla(s)",              "azul(es)"
         ),
         'gemConditions' => array(                           // ENCHANT_CONDITION_*
-            2 => ["menos de %d %s gema", "menos de %d %s gemas"],
```
Comment (Sarjuuk) [inline localization/locale_eses.php]: and here

File: localization/locale_eses.php [hunk 3]
```diff
@@ -379,25 +379,25 @@
     ),
     'mail' => array(
         'tokenExpires'  => "Este token expira en %s",
-        'accConfirm'    => ["Confirmación de Cuenta", "Bienvenido a ".CFG_NAME_SHORT."!\r\n\r\nHaga click en el enlace siguiente para activar su cuenta.\r\n\r\n".HOST_URL."?account=signup&token=%s\r\n\r\nSi usted no solicitó este correo, por favor ignorelo."],
```
Comment (AFROM) [inline localization/locale_eses.php]: Oops :( my bad

File: localization/locale_eses.php (hunk 2 above)
Comment (AFROM) [inline localization/locale_eses.php]: fixing

**Pull request merged**